### PR TITLE
feat: preconditions guide

### PR DESCRIPTION
**Pull request by kaname-png**

### CODE REVIEW

File: docs/Guide/preconditions/handling-permissions.mdx [hunk 1]
```diff
@@ -3,4 +3,91 @@ sidebar_position: 2
...
+
+```javascript {4-11}
+const { Command } = require('@sapphire/framework');
+
+module.exports = class BanCommand extends Command {
+  constructor(context, options) {
+    super(context, {
+      ...options,
+      name: 'ban',
+      aliases: ['ban'],
```
Comment (UndiedGamer) [inline docs/Guide/preconditions/handling-permissions.mdx:38]: what

File: docs/Guide/preconditions/nsfw-filter.mdx [hunk 2]
```diff
@@ -3,4 +3,72 @@ sidebar_position: 1
...
+
+```javascript {4-11}
+import { Command } from '@sapphire/framework';
+
+export class LewdCommand extends Command {
+  constructor(context, options) {
+    super(context, {
+      ...options,
+      name: 'lewd',
+      aliases: ['lewd'],
```
Comment (UndiedGamer) [inline docs/Guide/preconditions/nsfw-filter.mdx:45]:
```suggestion
```

Comment (UndiedGamer) [reply docs/Guide/preconditions/nsfw-filter.mdx:45]: do this for everything

**Review by favna — CHANGES_REQUESTED** [review]
Superseded by #91

**Pull request closed without merging**